conference/react-js/src/components/App.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import {
+  BrowserRouter as Router,
+  Route,
+} from 'react-router-dom';
+import ForgotPassword from './../containers/Forgot-password';
+import Tabs from './Tabs';
+import Header from './Header';
+import '../scss/main.scss';
+import SignUp from '../containers/SignUpForm';
+
+const App = () => (
+  <Router>
+    <div>
+      <Header />
```

Comment:
maybe, there is sense to move Header out for Router?

Reply:
We have some link in header for control in router

Reply:
ok then)